includes/evf-entry-functions.php

```diff
@@ -19,7 +19,7 @@
 function evf_get_entry( $id, $with_fields = false, $args = array() ) {
 	global $wpdb;
 
-	if ( ! isset( $args['cap'] ) ) {
+	if ( ! isset( $args['cap'] ) && is_admin() ) {
```

Comment:
@milan1750 I think we have to sync the forms way over here too. Can you also test if we add `wp_doing_ajax()` wouldn't conflict?